src/clj/game/cards/identities.clj

```diff
@@ -1644,20 +1644,14 @@
 
 (defcard "Weyland Consortium: Built to Last"
   {:events [{:event :advance
-             :optional
-             {:req (req (not (pos? (- (get-counters target :advancement) (:amount (second targets) 0)))))
-              :prompt "Gain 2 [Credits]?"
-              :autoresolve (get-autoresolve :auto-build-to-last)
-              :yes-ability
-              {:async true
-               :msg "gain 2 [Credits]"
-               :effect (req (gain-credits state :corp eid 2))}}}]
-   :abilities [(set-autoresolve :auto-build-to-last "Built to Last")]})
+             :req (req (not (pos? (- (get-counters target :advancement) (:amount (second targets) 0)))))
+             :msg "gain 2 [Credits]"
+             :effect (req (gain-credits state :corp eid 2))}]})
```

Comment:
This ability needs `:async true`

Reply:
Easy enough - probably a bit much for here, but the doc says
```clojure 
;  :async -- boolean
;   Mark the ability as "async", meaning the :effect function must call effect-completed itself.
;   Without this being set to true, resolve-ability will call effect-completed once it's done.
;   This part of the engine is really dumb and complicated, so ask someone on slack about it.```
 Looks like the most relevant code is `engine/do-ability`?